modules/apps/configuration-admin/configuration-admin-web/src/main/java/com/liferay/configuration/admin/web/internal/category/GooglePlacesConfigurationCategory.java

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.configuration.admin.web.internal.category;
+
+import com.liferay.configuration.admin.category.ConfigurationCategory;
+
+import org.osgi.service.component.annotations.Component;
+
+/**
+ * @author Renato Rego
+ */
+@Component(service = ConfigurationCategory.class)
+public class GooglePlacesConfigurationCategory
```

Comment:
For sure, @rodrigopaulino! Analyzing the code, I realized that to create a dedicated category in the settings (Instance and Site) to hold the Google Places settings would be necessary create it on the module `configuration-admin-web`. :slightly_smiling_face:

Reply:
Thanks for answering, @natocesarrego.

If that was your motivation, don't worry. There's no technical limitation in creating the new ConfigurationCategory inside the `google-places-web` module. Since this is an OSGi component and the service you're implementing is not private, the new category will be available just as you want, even from a different module. 😃 

Btw, before the Language.properties centralization, I believe you'd have to override the `getBundleSymbolicName` method in order to place your language keys inside the `google-places-web` module, but I don't think that's relevant anymore.

Please, be sure to let me know if you have questions about this. I'll be happy to answer them.

Reply:
Ok, @rodrigopaulino, I'll give it a try and see how it goes. :wink: